docs/admin/install/serving/ingress-implementations.md

```diff
@@ -0,0 +1,11 @@
+# Unofficial Ingress Implementations
+
+In addition to the Knative Ingress implementations in the ([install a networking layer](../install-serving-with-yaml/#install-a-networking-layer)) section,
+there are other Knative Ingress implementations.
```

Comment:
```suggestion
In addition to the Knative ingress implementations in the ([install a networking layer](../install-serving-with-yaml/#install-a-networking-layer)) section, there are other Knative ingress implementations.
```

This isn't capitalized below, so I'd decide on either caps or no caps and then keep it consistent. Unless it's a literal API object we're referring to I'd recommend no caps.